sandboxes/with-perps/cannonfile.toml

```diff
@@ -0,0 +1,622 @@
+fromCall.func = "owner"
+func = "setLockedOiRatio"
+args = [
+    "<%= extras.perps_eth_market_id %>", # uint128 marketId
+    "<%= parseEther('0.5') %>"
+    # uint256 lockedOiRatioD18
+]
+
+[invoke.configure_max_collateral_amount_for_perps_eth]
+target = ["perps_factory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "setMaxCollateralAmount"
+args = [
+    "<%= extras.synth_eth_market_id %>", # uint128 synthMarketId
+    "<%= MaxUint256 %>"
+    # uint256 collateralAmount
+]
+
+#
+#
+# Global Perps configuration
+#
+#
+[invoke.configure_perps_liquidation_reward_limits]
+target = ["perps_factory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "setLiquidationRewardGuards"
+args = ["<%= parseEther('1') %>", "<%= parseEther('500') %>"]
+
+[invoke.configure_perps_max_collateral_for_susd]
+target = ["perps_factory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "setMaxCollateralAmount"
+args = ["0", "<%= MaxUint256 %>"]
+
+[invoke.configure_perps_synth_deduction_priority]
+target = ["perps_factory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "setSynthDeductionPriority"
+args = [
+    [
+        "<%= extras.synth_box_market_id %>",
+        "<%= extras.synth_eth_market_id %>",
+        #        "<%= extras.synth_btc_market_id %>",
+        "0"
+    ]
+]
+
+[invoke.configure_per_account_caps_perps]
+target = ["perps_factory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "setPerAccountCaps"
+args = ["5", "3"]
+
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+#
+# Create Spartan Council pool
+#
+#
+[invoke.permit_owner_to_create_pools]
+target = ["synthetix.CoreProxy"]
+fromCall.func = "owner"
+func = "addToFeatureFlagAllowlist"
+args = ["<%= formatBytes32String('createPool') %>", "<%= settings.owner %>"]
+
+[invoke.create_spartan_council_pool]
+target = ["synthetix.CoreProxy"]
+func = "createPool"
+fromCall.func = "owner"
+args = ["1", "<%= settings.owner %>"]
+extra.spartan_council_pool_id.event = "PoolCreated"
+extra.spartan_council_pool_id.arg = 0
+depends = ["invoke.permit_owner_to_create_pools"]
+
+[invoke.configure_spartan_council_pool_name]
+target = ["synthetix.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= extras.spartan_council_pool_id %>"]
+func = "setPoolName"
+args = ["<%= extras.spartan_council_pool_id %>", "Spartan Council Pool"]
+
+[invoke.make_spartan_council_pool_a_preferred_pool]
+target = ["synthetix.CoreProxy"]
+fromCall.func = "owner"
+func = "setPreferredPool"
+args = ["<%= extras.spartan_council_pool_id %>"]
+
+[invoke.configure_spartan_council_pool]
+target = ["synthetix.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= extras.spartan_council_pool_id %>"]
+func = "setPoolConfiguration"
+args = [
+    "<%= extras.spartan_council_pool_id %>",
+    # TODO: for some reason all of these markets fail here with "Market does not exist" error. Need to figure it out
```

Comment:
This issue is reproduced here https://github.com/Synthetixio/synthetix-sandbox/pull/7
Something to do with `salt` apparently.

Reply:
This is solved by importing external packages instead of provisioning them